www/artists/thall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Even ignoring the obvious fact that this file shouldn't be here because it's irrelevant to this issue, why does Don Hall have files in a folder named thall? This is exactly the same as the index.html in dhall. Why does he get two copies of his page?